cmd/ebrelayer/relayer/network_test.go

```diff
@@ -0,0 +1,31 @@
+package relayer
+
+// ------------------------------------------------------------
+//    Network_test
+//
+//    Tests network.go functionality.
+//
+// ------------------------------------------------------------
+
+import (
+  "testing"
+  "fmt"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+  Client = "wss://ropsten.infura.io/ws"
```

Comment:
Ideally we don't want to depend on infura for our test suite/CI - should rather have a local blockchain running. But think this is ok for now

Reply:
Agreed. We should use truffle (or the library it's leveraging) to deploy a local Ethereum blockchain for this.